queue.js

```diff
@@ -263,6 +263,17 @@ const queue = {
     if (fs.existsSync(cache_filename)) {
       var raw_data = fs.readFileSync(cache_filename);
       levels = JSON.parse(raw_data);
+      const username_missing = level => !level.hasOwnProperty('username');
+      if (levels.some(username_missing)) {
+        console.warn(`Usernames are not set in the file ${cache_filename}!`);
+        console.warn('Assuming that usernames are lowercase Display Names which does work with Localized Display Names.');
+        console.warn('To be save clear the queue with !clear.');
```

Comment:
```suggestion
        console.warn('To be safe, clear the queue with !clear.');
```

Reply:
Thank you for correcting my spelling mistake.